sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -383,6 +383,13 @@ class DeviceBinaryImage {
     DeviceGlobals.init(Bin, __SYCL_PI_PROPERTY_SET_SYCL_DEVICE_GLOBALS);
     return DeviceGlobals;
   }
+  const PropertyRange getHostPipes() const {
+    // We can't have this variable as a class member, since it would break
+    // the ABI backwards compatibility.
```

Comment:
This do-not-break-my-ABI story starts being a technical debt generator... :-(

Reply:
> @keryell, I'm aware of that problem. I think we should start tracking these in terms of tasks for the future, when we can break old ABI compatibility. @s-kanaev, do we track technical debt caused by ABI compatibility requirement?

I know @pvchupin has a link where it is tracked

Reply:
Thanks! I will add `//TODO` in a separate commit that annotated all places in this files that will require changes after ABI policy change.

Reply:
Updated the list.

Reply:
The TODOs are annotated in #5929